tests/test_setup_config_extended.py

```
@@ -332,6 +332,43 @@ def test_setup_downloads_apk_snapshot_enabled(mocker):
+@pytest.mark.unit
+def test_setup_client_app_prompts_for_snapshot_retention(monkeypatch):
+    config = {
+        "SAVE_CLIENT_APPS": True,
+        "SAVE_APKS": True,
+        "SELECTED_APP_ASSETS": ["androidApp-google-universal-debug-*.apk"],
+        "CHECK_APP_SNAPSHOTS": True,
+        "APP_SNAPSHOT_VERSIONS_TO_KEEP": 1,
+    }
+    monkeypatch.setattr("builtins.input", lambda _prompt="": "3")
+
+    result = setup_config._setup_client_app(config, False, 3)
```

Ruff (0.16.0)

[warning] Boolean positional value in function call

(FBT003)